README.md

```diff
@@ -17,11 +17,11 @@ Go to the [wiki](https://github.com/Ikuyadeu/vscode-R/wiki) to view the document
     install.packages(c("jsonlite", "rlang"))
     ```
 
-3. Install the [R extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=Ikuyadeu.r).
+3. Install the [R extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=REditorSupport.r).
```

Comment:
Since the Publisher is the same, it won't be changed like https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode ?

Reply:
Sorry my mistake.